models/aws_cloud_cost__overview.sql

```diff
@@ -72,7 +72,7 @@ fields as (
 
     select 
         source_report.source_relation,
-        report, 
+        report,  
```

Comment:
I noticed this is `null` for all our internal data. Any idea why?

Reply:
Also, can we add aliases to each field we are selecting in this model. There is a lot to track here and it isn't entirely clear where each of these fields is coming from. Adding the alias will really help provide transparency regarding where each field is coming from.

Reply:
regex was messed up, should be fixed now! 

what sorts of aliases are you thinking? I was kinda leaning away from aliases to make the mapping to AWS's column [docs](https://docs.aws.amazon.com/cur/latest/userguide/table-dictionary-cur2-bill.html) as straightforward as possible

Reply:
actually walking this back.... aliasing them now and it's so much nicer